kolibri_explore_plugin/assets/src/views/ExploreIndex.vue

```diff
@@ -1,26 +1,10 @@
 <template>
 
-  <CoreBase
-    :marginBottom="bottomSpaceReserved"
-    :authorized="userIsAuthorized"
-    authorizedRole="registeredUser"
-    v-bind="immersivePageProps"
-    :maxMainWidth="Infinity"
-  >
-    <template v-if="pageName !== 'TOPICS_ROOT'" slot="app-bar-actions">
-      <router-link
-        class="rm-link-style"
-        :style="{ color: $themeTokens.textInverted }"
-        to="/"
-      >
-        Back
-      </router-link>
-    </template>
-
+  <BaseComponent :back="pageName !== 'TOPICS_ROOT'">
     <!--
-      Topics pages have a different heading style which
-      includes passing the breadcrumbs
-    -->
+        Topics pages have a different heading style which
+        includes passing the breadcrumbs
+      -->
```

Comment:
Is this change because of lint? Strange that the `<!--` is not indented.

Reply:
Yes, this is a linter change, but it's not needed, I had some problems with the `Base` component not been detected by the linter so this code gets updated and I didn't change back once I resolved the issue.